build.gradle

```
@@ -2,6 +2,9 @@ plugins {
     id 'java'
     id 'org.springframework.boot' version '3.5.3'
     id 'io.spring.dependency-management' version '1.1.7'
+
+    //이 버전을 관리하면 아래의 dependencies의 버전을 관리한다.
+    //즉 springframework에 관련된 애들의 버전을 저 맨위 있는것으로 관리한다.
```

주석으로 남기신 이유가 있을까요? 
참고로 주석의 위치는 주석의 내용의 대상이 되는 곳에 남기는 게 좋습니다!
```suggestion
    id 'java'
    id 'org.springframework.boot' version '3.5.3'
    // 이 버전을 관리하면 아래의 dependencies의 버전을 관리한다.
    // 즉 springframework에 관련된 애들의 버전을 저 맨위 있는것으로 관리한다.
    id 'io.spring.dependency-management' version '1.1.7'
```

혹은 `id 'io.spring.dependency-management' version '1.1.7' // 이 버전을 관리하면 ...` 이런 형태도 있어요!